src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/InitiationTaskHandlerTest.java

```diff
@@ -70,12 +70,19 @@ void evaluateDmn() {
         EvaluateDmnRequest<InitiateEvaluateRequest> requestParameters =
             InitiateTaskHelper.buildInitiateTaskDmnRequest();
 
-        Mockito.when(apiClientToInitiateTask.evaluateDmn(DMN_NAME, requestParameters))
-            .thenReturn(new EvaluateDmnResponse<>(Collections.emptyList()));
+        Mockito.when(apiClientToInitiateTask.evaluateDmn(
+            DMN_NAME,
+            requestParameters,
+            TENANT_ID
```

Comment:
What happens if tenant_id is not provided or is blank? Do we need a test to cover this?